### ISSUE TITLE
Support For PreferenceFragment / PreferenceActivity Is Missing

### ISSUE THREAD

**Issue by JustinTipton** [grandcentrix/tray#67]
If the goal of this project is to be a SharedPreferences replacement, it should support PreferenceFragment / PreferenceActivity, but I don't see that ability.

**Comment by proninyaroslav** [comment]
Is there a workaround?

**Comment by StefMa** [comment]
I don't think so, that we will create a `TrayActivity` or `TrayFragment` in the future.

To catch all automatic saved values from these Classes you can add a `OnSharedPreferenceChangeListener`.
For example:
```java
@Override
public void onResume() {
    super.onResume();
    getPreferenceManager().getSharedPreferences()
       .registerOnSharedPreferenceChangeListener(this);
}
```

**Comment by StefMa** [comment]
Hey guys,

there is an community project of this.
Haven't yet looked into it, but maybe it helps?!

See #82

**Comment by draekko** [comment]
@StefMa It replaces most of the views and the activity/fragments and wraps it around Tray instead of SharedPreferences. I use it in one app as UI replacement for all that SharedPreferences crap which doesn't work (well or at all) with multiprocess apps.

**Comment by draekko** [comment]
@StefMa  Updated the code so now it should now work fine for API 21+ and added missing notify listeners.
